internal/web/validate.go

```diff
@@ -410,10 +421,11 @@ func runValidatorBoth(validator, repopath, commit, commitname string, gcl *gincl
 
 		glog.Init()
 		clonechan := make(chan git.RepoFileStatus)
+		pth, _ := os.Getwd()
 		os.Chdir(tmpdir)
 		go gcl.CloneRepo(repopath, clonechan)
 		for stat := range clonechan {
-			if stat.Err != nil {
+			if stat.Err != nil && stat.Err.Error() != "Error initialising local directory" {
```

Comment:
The execution always prints out "Error initialising local directory", but even that the error occurs, it otherwise works OK